### ISSUE TITLE
Add option to set the duration of animations (or remove animation)

### ISSUE THREAD

**Issue by Gbuomprisco** [Gbuomprisco/ngx-chips#477]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => search github for a similar issue or PR before submitting
[ x ] feature request
[ ] support request/question
```

**Current behavior**

Animations are not configurable

**Expected behavior**

Allow more configuration, many people don't like the animation

**Comment by Gbuomprisco** [comment]
https://github.com/Gbuomprisco/ngx-chips/pull/476/

**Comment by Gbuomprisco** [comment]
published v1.4.2
